src/imgix-core-js.js

```diff
@@ -174,10 +174,16 @@
         targetWidths = this._generateTargetWidths(widthTolerance, minWidth, maxWidth);
       }
 
+      var queryParams = {};
+      var key, value;
```

Comment:
I think value is unused here

Reply:
Yeah, you're right, good catch! I wrote this in two places (L197, L177), but forgot to remove the `value` var here. Let me clean this up.